public/index.html

```diff
@@ -411,14 +414,10 @@ <h5 class="modal-title" id="exampleModalLongTitle">Terms and Conditions</h5>
 			<div class="row homepageFullpage">
 				<div class="col land-splash pr-0 pl-0 text-center d-flex justify-content-center align-items-center">
 					<div class="d-flex flex-column text-left text-white p-5">
-						<h1>Create an online storefront</h1>
```

Comment:
were you using an outdated branch when you made this? these changes were from a while ago when Emile requested them

Reply:
I'm so confused, I didnt even make most of these changes, and this was a fresh branch..
The only thing this branch was supposed to do was remove the modal header so the line overlap in login would disappear.

Reply:
really weird idek how that happened 😆 maybe just manually put it back to how it was and recommit 🤷‍♂